### PR TITLE
Fix test warnings and assert_raise failures

### PR DESCRIPTION
**Pull request by jva**
Method assert_raises() does not work with error subclasses. Used workaround from https://stackoverflow.com/questions/20617670/how-to-get-assert-raise-to-handle-exception-subclasses

Added gem "minitest" as suggested by rake warnings.

### CODE REVIEW

File: test/toml_test.rb [hunk 1]
```diff
@@ -117,8 +117,10 @@ def test_valid_cases
   def test_invalid_cases
     file = '*'
     Dir["test/examples/invalid/#{file}.toml"].each do |toml_file|
-      assert_raises(TomlRB::Error, "For file #{toml_file}") do
+      begin
```
[the remark this answers is not in the document]
Comment (jva) [reply test/toml_test.rb]: I don't see how this is redundant. rake fails if "begin" is removed.

File: test/toml_test.rb [hunk 2]
```diff
@@ -117,8 +117,10 @@ def test_valid_cases
...
         TomlRB.load_file(toml_file)
+      rescue => e
```
[the remark this answers is not in the document]
Comment (jva) [reply test/toml_test.rb]: Class checking is done inside rescue block.

File: test/helper.rb [hunk 3]
```diff
@@ -1,2 +1,3 @@
+gem "minitest"
```
[the remark this answers is not in the document]
Comment (jva) [reply test/helper.rb]: Not in scope

**Comment by emancu** [comment]
@jva I don't see any warnings or how this was a problem. Is this PR still relevant?

**Comment by emancu** [comment]
I'm closing this because it I never got a reply. Feel free to reopen @jva

**Pull request closed without merging**